### PR TITLE
feat: support /uptime

### PR DESCRIPTION
**Pull request by eliskeggs-asana**
This endpoint returns the time since the process started.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "tsc",
-    "prepare": "npm run build"
+    "prepublishOnly": "npm run build"
```
Comment (mmorearty) [inline package.json:17]: ^ What is this about?

Comment (eliskeggs-asana) [reply package.json:17]: Ah, sorry - meant to note that. `prepare` as a hook makes it run when you do anything like `npm install` or `npm ci`, which is a bit of a strange workflow. On second thought, it doesn't belong in this PR.

File: src/server.ts [hunk 2]
```diff
@@ -119,6 +119,7 @@ function pathToUploadCache(s3key: string, config: Config) {
 
 export function startServer(s3: AWS.S3, config: Config, onDoneInitializing: () => void) {
     const cache = new Cache(config); // in-memory cache
+    const serverStartTime = Math.floor(Date.now() - (process.uptime() * 1000));
```
Comment (mmorearty) [inline src/server.ts:122]: Could you please put units in the variable name, for clarity? E.g. `serverStartTimeMs` or `serverStartTimeMillis` or whatever

File: src/server.ts [hunk 3]
```diff
@@ -206,6 +207,8 @@ export function startServer(s3: AWS.S3, config: Config, onDoneInitializing: () =
             case "GET": {
                 if (s3key === "ping") {
                     sendResponse(req, res, "pong", { startTime, awsPaused });
+                } else if (s3key === "uptime") {
+                    sendResponse(req, res, String(Date.now() - serverStartTime), { startTime, awsPaused });
```
Comment (mmorearty) [inline src/server.ts:211]: Could you please add a comment saying "returns uptime in milliseconds"

**Review by mmorearty — CHANGES_REQUESTED** [review]

File: src/server.ts (hunk 2 above)
Comment (eliskeggs-asana) [reply src/server.ts]: 5b2683b

File: src/server.ts (hunk 3 above)
Comment (eliskeggs-asana) [reply src/server.ts]: 5b2683b

**Pull request closed without merging**